### DISCUSSION TITLE
Unwanted 2Fa

### DISCUSSION THREAD

**Discussion by rogercreagh** [community/community#66214] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Firstly to say that I find your documentation on 2FA almost incomprehensible - it seems to assume a depth of knowledge about "security" that I simply don't have or need, and is full of unexplained jargon (eg what  is TOTP which seems to get mentioned a lot in the basic intro but never explained in terms that you can understand if you don't already know what it is - many other examples like this), and it seems that the methods that avoid having to have a second device to hand are simply not available on Linux Ubutnu which I use for development.

Secondly as a non-professional developer I regard the whole security thing across multiple areas as a massive confidence trick to obfuscate our simple human responsibility to look out for ourselves and each other. In almost all cases it introduces complexity and inconvenience for no real benefit to the user - and for no stated benefit to the provider. 2FA on GitHub certainly falls into this category.

Since is is a massive dis-benefit to the individual user, where is a simple statement of what the benefit to GitHub Inc is? Is there any benefit to the organisation? If you can't explain what it is then perhaps you can explain what the benefit to the end user (me) is supposed to be.  

One basic question - if I stop contributing code on GitHub can I then continue to use a simple password access account? I think it is going to be less hassle for me to simply migrate to using git on my own server for my own code. I guess I will still be able to see public repositories here without having to log in, but it would be useful to have a simple account that enabled me to report bugs and contribute to discussions on repositories - is this possible?

It is a great shame that GitHub has fallen into the 2FA hype - but I guess I must be in a minority in finding it of no use whatsoever?

**Comment by howtophil** [comment, marked as answer]
I've started moving to bitbucket since they aren't pulling 2FA requirement nonsense.

2FA is not more secure than a strong password.

**Reply by kuroodo** [reply]
I've seen a lot of people recommend and migrate to bitbucket. I want to add that GitLab is also a solid choice for anyone wanting options

**Reply by rogercreagh** [reply]
Thanks both @howtophil  and @kuroodo . I'll certainly look into bitbucket - I've got three weeks left. I was considering GitLab community edition running on my own VPS.

**Reply by justin-octo** [reply]
There are several high quality options.

**Reply by justin-octo** [reply]
https://github.com/howtophil/No2FA
Fork it to show your support

**Reply by justin-octo** [reply]
I stood up my own local Gitea docker/podman container and pulled all my repos down from GitHub, and then pushed them up to my new server.  So, I have a local copy.  Sucks that Microsoft ruins good things like this.

**Comment by avinmaster** [comment]
Have they canceled the 2FA requirement for all users, or is it only required for people, who contribute to GitHub community repos?

**Reply by ghost** [reply]
not at all!!! i use github for backups and school projects only, for less than a year!! and i'm already getting my second account blocked

(they've already blocked me from accessing my first account)
